prow/config.yaml

```diff
@@ -166,6 +166,11 @@ presets:
           secretKeyRef:
             name: kyma-bot-github-token
             key: token
+      - name: GITHUB_TOKEN
```

Comment:
Is this duplication in preset needed. What do you think about setting this value during job execution?